qldbdriver/transaction.go

```diff
@@ -25,9 +25,9 @@ import (
 // Transaction represents an active QLDB transaction.
 type Transaction interface {
 	// Execute a statement with any parameters within this transaction.
-	Execute(statement string, parameters ...interface{}) (*Result, error)
-	// Buffer a Result into a BufferedResult to use outside the context of this transaction.
-	BufferResult(result *Result) (*BufferedResult, error)
+	Execute(statement string, parameters ...interface{}) (Result, error)
+	// Buffer a qldbResult into a BufferedResult to use outside the context of this transaction.
```

Comment:
This docstring should not be changed. The user of the docs has no idea, and shouldn't, what a "qldbResult" is.

Reply:
Addressed.